ui/stories/icon.stories.js

```diff
@@ -3,9 +3,11 @@ import hbs from 'htmlbars-inline-precompile';
 import { storiesOf } from '@storybook/ember';
 import notes from './icon.md';
 import icons from '../node_modules/@hashicorp/structure-icons/dist/index.js';
+import { withKnobs, select } from '@storybook/addon-knobs';
 
 storiesOf('Icon/', module)
-  .addParameters({ options: { showPanel: false } })
+  .addParameters({ options: { showPanel: true} })
```

Comment:
Not sure if you left this in by accident (`showPanel: true`)? Ah, maybe you want the panel in now as you have a setting you can tweak.? Will leave this comment in just incase, but 99% sure this is good

Reply:
Yeah, this is so that the knobs panel shows up when you click Icon in the sidebar, otherwise it's hidden. We usually hide it if we're not using the knobs decorator, but since I added it here, I flipped it to `true`.

Reply:
Sweet gotcha 👍